client/app/assets/less/main.less

```diff
@@ -1,3 +1,8 @@
+@import '~antd/lib/style/core/iconfont.less';
+@import '~antd/lib/input/style/index.less';
+@import '~antd/lib/date-picker/style/index.less';
+@import 'redash/redash-ant';
```

Comment:
It's better to keep the component specific stylesheets imports next to the components, so if we remove them we remove everything together.

Reply:
Ok, if I move it back to the component, what's the correct path for 'redash/redash-ant'; ?

Reply:
I think we should keep it where it is, and import per-component:
```javascript
import 'antd/lib/style/core/iconfont.less';
import 'antd/lib/input/style/index.less';
import 'antd/lib/date-picker/style/index.less';
import '@/assets/redash/redash-ant.less';
```

@arikfr - WDYT?

Reply:
Is the order of imports important? Because of not, we can keep `redash-ant.less` in `main.less` to avoid repeating it everywhere.

Reply:
Also, it's a small thing, but `redash/redash-ant.less` feels repetitive... I think that naming it `ant.less` is enough.

Reply:
redash-ant.less is renamed to ant.less, makes sense.

The order of imports seems to be important otherwise it won't work.

Reply:
The order of import is even more important:

import '@/assets/less/redash/ant.less';
import 'antd/lib/style/core/iconfont.less';
import 'antd/lib/input/style/index.less';
import 'antd/lib/date-picker/style/index.less';

This works 👌

Reply:
Ok, I tried these things and a variety of others but those won't work. The only thing worked when it was placed in the first lines of main.less. Maybe I'm missing something?